File: app/src/main/java/com/aircraft/toolmanagment/data/entity/Tool.java

package com.aircraft.toolmanagment.data.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tools")
public class Tool {
    @PrimaryKey(autoGenerate = true)
    public int id;
    public String name;
    public String model;
    public String specification;
    public int quantity;
    public String manufacturer;
    public long purchaseDate;
    public String storageLocation;
    public String barcode;
    public String status;
    public long updatedAt;

    public Tool() {}
}

File: app/src/main/java/com/aircraft/toolmanagment/data/entity/Tool.kt
package com.aircraft.toolmanagment.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tools")
data class Tool(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val name: String = "",
    val model: String = "",
    val specification: String = "",
    val quantity: Int = 0,
    val manufacturer: String = "",
    val purchaseDate: Long = 0,
    val storageLocation: String = "",
    val barcode: String = "",
    val status: String = "",
    val updatedAt: Long = 0
)